documentation/modules/auxiliary/scanner/portscan/tcp.md

```diff
@@ -0,0 +1,69 @@
+## Description
+  
+   This module will Enumerate open TCP services by performing a full TCP     connect on each port. This will establish a complete three-way handshake (SYN -> SYN/ACK -> ACK) on the target port. This does not need administrative privileges on the sourcemachine, which may be useful if pivoting.
```

Comment:
Remove the extra spaces between TCP and connect.
Lower case E on enumerate.
Add space between source and machine.